.github/workflows/ci.yaml

```diff
@@ -30,7 +30,9 @@ jobs:
         run: yarn run test:unit && yarn run test:components
 
       - name: Run e2e tests
-        run: yarn run test:e2e
+        run: |
+          yarn add cheerio-select
```

Comment:
eh?

Reply:
`--frozen-lockfile` dose not install `cheerio-select` even if is `cheerio-select` part of `package.json`

Reply:
if it's in lockfile it must be installed

Reply:
I have no idea why, maybe you can figure it out?